swarm_dial.go

```diff
@@ -237,18 +237,18 @@ func (s *Swarm) DialPeer(ctx context.Context, p peer.ID) (network.Conn, error) {
 // dialbackoff.
 func (s *Swarm) dialPeer(ctx context.Context, p peer.ID) (*Conn, error) {
 	log.Debugf("[%s] swarm dialing peer [%s]", s.local, p)
-	var logdial = lgbl.Dial("swarm", s.LocalPeer(), p, nil, nil)
+	lgbl.Dial("swarm", s.LocalPeer(), p, nil, nil)
```

Comment:
This doesn't actually do anything. We should replace these two lines with:

```go
log.Debugw("dialing peer", "from", s.local, "to", p)
```

Reply:
Fixed in #255.